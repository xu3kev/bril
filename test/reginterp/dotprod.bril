main{
v0: int = const 5;
v1: int = const 2;
v2: int = const 6;
v3: int = const 3;
v4: int = const 4;
v5: int = const 8;
v6: int = const 1;
v7: int = const 8;
v8: int = const 1;
v9: int = const 7;
v10: int = const 2;
v11: int = const 8;
v12: int = const 3;
v13: int = const 1;
v14: int = const 3;
v15: int = const 1;
v16: int = const 0;
v17: int = mul v0 v8;
v16: int = add v16 v17;
v18: int = mul v0 v9;
v16: int = add v16 v18;
v19: int = mul v0 v10;
v16: int = add v16 v19;
v20: int = mul v0 v11;
v16: int = add v16 v20;
v21: int = mul v0 v12;
v16: int = add v16 v21;
v22: int = mul v0 v13;
v16: int = add v16 v22;
v23: int = mul v0 v14;
v16: int = add v16 v23;
v24: int = mul v0 v15;
v16: int = add v16 v24;
v25: int = mul v1 v8;
v16: int = add v16 v25;
v26: int = mul v1 v9;
v16: int = add v16 v26;
v27: int = mul v1 v10;
v16: int = add v16 v27;
v28: int = mul v1 v11;
v16: int = add v16 v28;
v29: int = mul v1 v12;
v16: int = add v16 v29;
v30: int = mul v1 v13;
v16: int = add v16 v30;
v31: int = mul v1 v14;
v16: int = add v16 v31;
v32: int = mul v1 v15;
v16: int = add v16 v32;
v33: int = mul v2 v8;
v16: int = add v16 v33;
v34: int = mul v2 v9;
v16: int = add v16 v34;
v35: int = mul v2 v10;
v16: int = add v16 v35;
v36: int = mul v2 v11;
v16: int = add v16 v36;
v37: int = mul v2 v12;
v16: int = add v16 v37;
v38: int = mul v2 v13;
v16: int = add v16 v38;
v39: int = mul v2 v14;
v16: int = add v16 v39;
v40: int = mul v2 v15;
v16: int = add v16 v40;
v41: int = mul v3 v8;
v16: int = add v16 v41;
v42: int = mul v3 v9;
v16: int = add v16 v42;
v43: int = mul v3 v10;
v16: int = add v16 v43;
v44: int = mul v3 v11;
v16: int = add v16 v44;
v45: int = mul v3 v12;
v16: int = add v16 v45;
v46: int = mul v3 v13;
v16: int = add v16 v46;
v47: int = mul v3 v14;
v16: int = add v16 v47;
v48: int = mul v3 v15;
v16: int = add v16 v48;
v49: int = mul v4 v8;
v16: int = add v16 v49;
v50: int = mul v4 v9;
v16: int = add v16 v50;
v51: int = mul v4 v10;
v16: int = add v16 v51;
v52: int = mul v4 v11;
v16: int = add v16 v52;
v53: int = mul v4 v12;
v16: int = add v16 v53;
v54: int = mul v4 v13;
v16: int = add v16 v54;
v55: int = mul v4 v14;
v16: int = add v16 v55;
v56: int = mul v4 v15;
v16: int = add v16 v56;
v57: int = mul v5 v8;
v16: int = add v16 v57;
v58: int = mul v5 v9;
v16: int = add v16 v58;
v59: int = mul v5 v10;
v16: int = add v16 v59;
v60: int = mul v5 v11;
v16: int = add v16 v60;
v61: int = mul v5 v12;
v16: int = add v16 v61;
v62: int = mul v5 v13;
v16: int = add v16 v62;
v63: int = mul v5 v14;
v16: int = add v16 v63;
v64: int = mul v5 v15;
v16: int = add v16 v64;
v65: int = mul v6 v8;
v16: int = add v16 v65;
v66: int = mul v6 v9;
v16: int = add v16 v66;
v67: int = mul v6 v10;
v16: int = add v16 v67;
v68: int = mul v6 v11;
v16: int = add v16 v68;
v69: int = mul v6 v12;
v16: int = add v16 v69;
v70: int = mul v6 v13;
v16: int = add v16 v70;
v71: int = mul v6 v14;
v16: int = add v16 v71;
v72: int = mul v6 v15;
v16: int = add v16 v72;
v73: int = mul v7 v8;
v16: int = add v16 v73;
v74: int = mul v7 v9;
v16: int = add v16 v74;
v75: int = mul v7 v10;
v16: int = add v16 v75;
v76: int = mul v7 v11;
v16: int = add v16 v76;
v77: int = mul v7 v12;
v16: int = add v16 v77;
v78: int = mul v7 v13;
v16: int = add v16 v78;
v79: int = mul v7 v14;
v16: int = add v16 v79;
v80: int = mul v7 v15;
v16: int = add v16 v80;
print v16;
}
