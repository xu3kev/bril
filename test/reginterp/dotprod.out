962
#load: 257
#store: 145
